maxim/filter_objects.py

```
@@ -65,13 +69,30 @@ def parse_incoming_query(incoming_query: str) -> List[RuleType]:
                     field = field[2:]
+                parsed = False
                 # converting 'True'/'False':str to True/False:bool
                 if value == "True" or value == "False":
                     value = bool(value)
-                try:
-                    value = int(value)
-                except ValueError:
-                    pass
+                    parsed = True
+                if not parsed:
+                    if type(value) == bool:
```

Ruff (0.11.9)

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)